<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qrcode_time_reader</class>
 <widget class="QMainWindow" name="qrcode_time_reader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qrcode_time_reader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="photoPreview">
        <property name="text">
         <string>&lt;photo&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="fixTimes">
          <property name="text">
           <string>&amp;fix timestamps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="exifTime">
          <property name="text">
           <string>&lt;EXIF time&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="encodedTime">
          <property name="text">
           <string>&lt;encoded time&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeDifference">
          <property name="text">
           <string>&lt;time difference&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="photoList">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_photos"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionImport_photos">
   <property name="text">
    <string>Import photos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
